package.json

```diff
@@ -76,6 +74,7 @@
     "mocha": "^2.3.4",
     "moxios": "^0.2.0",
     "node-localstorage": "^1.3.0",
+    "react-stub-context": "^0.4.0",
```

Comment:
not needed since ``enzyme`` is able to inject context objects into component trees